common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -6190,6 +6190,12 @@
     ],
     "sqlState" : "42000"
   },
+  "UNSUPPORTED_TIME_PRECISION" : {
```

Comment:
Nit: UNSUPPORTED_TIME_SECONDS_PRECISION ? 
Though the precision part comes only from the seconds in the time, so I guess it's fine either way?

Reply:
IMHO, the error message already contains enough details including what you pointed out, and the condition name should comprise of minimum info which is enough to distinguish it from others. Also the error is about the `precision` parameter of the `TIME` type: 
```scala
case class TimeType(precision: Int)
```
from this point of view, it is self-descriptive.